nx/lib/nx/defn.ex

```diff
@@ -253,6 +253,15 @@ defmodule Nx.Defn do
 
         config :nx, :#{@app_key}, [compiler: EXLA, client: :cuda]
 
+  The function returns the values that were previously set as default
+  options.
+
+  ## Examples
+
+      iex> Nx.Defn.default_options(compiler: EXLA, client: :cuda)
+      []
```

Comment:
```suggestion
```

Reply:
We use a different compiler for tests. :)

Reply:
Ops, i was too naive. sorry, thanks for the correction 🙇